libnmstate/nm/sriov.py

```diff
@@ -129,9 +128,12 @@ def _remove_sriov_vfs_in_setting(vfs_config, sriov_setting, vf_ids_to_remove):
         yield vf_id
 
 
-def _has_sriov_capability(nm_client, ifname):
-    dev = device.get_device_by_name(nm_client, ifname)
-    return dev and (NM.DeviceCapabilities.SRIOV & dev.props.capabilities)
+def _has_sriov_capability(context, ifname):
+    dev = context.get_nm_dev(ifname)
+    if NM.DeviceCapabilities.SRIOV & dev.props.capabilities:
+        return True
+
+    return False
```

Comment:
I think the if statement format was requested by @EdDev. I prefer the one that you are using now.

Reply:
Restored to the old one.